webapp/channels/jest.config.js

```diff
@@ -30,7 +30,7 @@ const config = {
             '<rootDir>/src/packages/mattermost-redux/test/$1',
         '^mattermost-redux/(.*)$': '<rootDir>/src/packages/mattermost-redux/src/$1',
         '^.+\\.(jpg|jpeg|png|apng|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
-            'identity-obj-proxy',
+            '<rootDir>/src/tests/image_url_mock.json',
```

Comment:
The previous version of this made it so that importing any of the listed file types into a JS file would return an object, but that would cause React Testing Library to complain. Webpack is configured to make these imports return the URL of the file, so this is closer to how Webpack works